src/data/corpus_base.cpp

```diff
@@ -140,8 +252,7 @@ CorpusBase::CorpusBase(Ptr<Options> options, bool translate)
   }
 
   if(translate) {
-    ABORT_IF(vocabPaths.empty(),
-             "Translating, but vocabularies are not given!");
+    ABORT_IF(vocabPaths.empty(), "Translating, but vocabularies are not given!");
```

Comment:
Consider rewriting the error message. E.g.

Vocabularies must be given for translating.

Reply:
I will open a new PR with this change as it requires updates of regression tests.